sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c

```diff
@@ -105,7 +105,8 @@ void gnrc_ndp_internal_set_state(gnrc_ipv6_nc_t *nc_entry, uint8_t state)
             DEBUG("REACHABLE (reachable time = %" PRIu32 " us)\n", ipv6_iface->reach_time);
             t = ipv6_iface->reach_time;
 
-            /* we intentionally fall through here to set the desired timeout t */
+            /* Falls through. */
```

Comment:
in [1] (-Wimplicit-fallthrough=n) you can find a set of regexs that would match

[1] https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html